app/javascript/packs/baseTracking.js

```diff
@@ -0,0 +1,192 @@
+function initializeBaseTracking() {
+    console.log("*************Initializing base tracker from packs************************")
+    trackGoogleAnalytics3();
+    trackGoogleAnalytics4();
+    trackCustomImpressions();
+  }
+  
+  function trackGoogleAnalytics3() {
+    let wait = 0;
```

Comment:
Could you please fix this indentation detail? Starting on line 2 there's a "double tab" along the file all the way to the end.